### ISSUE TITLE
NEO template missing from Lookbook provisioning site

### ISSUE THREAD

**Issue by zemone24** [SharePoint/sp-provisioning-service#660]
Has this site template been decommissioned?  I had seen it the other day but now it does not show up on the lookbook site - all of the links via ms docs redirect to the main lookbook page and not to NEO templates.

**Comment by PaoloPia** [comment]
Hi,
Thanks for creating this issue. The NEO template has been split into three different templates:
- [NEW EMPLOYEE PRE-ONBOARDING SITE](https://lookbook.microsoft.com/details/8fefcc9a-7ca4-457f-bd10-acee3ae63b63)
- [NEW EMPLOYEE CORPORATE ONBOARDING SITE](https://lookbook.microsoft.com/details/388a159f-475b-4557-9088-c9073a26c576)
- [NEW EMPLOYEE DEPARTMENTAL ONBOARDING SITE](https://lookbook.microsoft.com/details/99d3b7ea-6ca0-430e-96bc-922d4d2de2ab)

The previous NEO unique template is now disabled. I'm going to close this issue. Thanks.

**Comment by zemone24** [comment]
Thank you!  When I tried each of the links before opening the ticket, they were all just going to the main lookbook page.  Appreciate your help!



________________________________

**Comment by zemone24** [comment]
Hi -

I wasn't sure how to re-open this issue but hoping you catch this.  I've successfully installed the Corporate onboarding and department onboarding, but the Pre-Onboarding template does not come up.  When I click the link you posted, it takes me to the main lookbook page.  Additionally, when I click on the Corporate Onboarding page, it has a link to the Pre-onboarding template.  That also takes me back to the main lookbook page.  I've tried multiple computers/browsers and same result.  Any suggestions?

Thanks
